lib/cloudformation.py

```diff
@@ -2008,3 +2011,19 @@ def add_event_rule(self, key, name, role_arn=None, schedule_expression=None, eve
         if description is not None:
             self.resources[key]["Properties"]["Description"] = description
 
+    def add_data_pipeline(self, key, name, objects, description="", depends_on=None):
```

Comment:
It'd be helpful to at least comment what objects is.

Reply:
Added comment for the method. Basically `objects` is built with `lib.datapipelne.DataPipeline()`.